torch/onnx/symbolic_opset11.py

```diff
@@ -944,3 +944,22 @@ def embedding_bag(g,
     # aten::embedding_bag returns a tuple of 4 elements: output, offset2bag, bag_size, max_indices.
     # But the last three outputs are not used in torch.nn.EmbeddingBag or torch.nn.functional.embedding_bag.
     return loop.node().output(), None, None, None
+
+
+def prim_ConstantChunk(g, self, chunks, dim):
```

Comment:
No, 
(1) `Slice` op here need dynamic input `start`, `end`, etc rather than attribute. So it needs opset 10+.
(2) `Slice-10`: axis can only be in range [0, r-1] which is rather restricted. So I prefer to support for opset 11+, as here.